core/variant.cpp

```diff
@@ -3411,8 +3588,8 @@ bool Variant::hash_compare(const Variant &p_variant) const {
 			return true;
 		} break;
 
-		case PACKED_REAL_ARRAY: {
-			hash_compare_pool_array(_data._mem, p_variant._data._mem, real_t, hash_compare_scalar);
+		case PACKED_FLOAT32_ARRAY: {
```

Comment:
64-bit missing here. But this might be what causes the crash reported in the PR.

Reply:
this functions seems like it was always broken, in fact it misses int32, I will fix it

Reply:
nevermind, forgot what this function was, fixing